intranet/static/js/eighth/responsive.js

```diff
@@ -83,9 +86,13 @@ function activityPickerResize() {
         var btnDown = $(".primary-content").hasClass("viewing") && w <= 500 ? 0 : 120; // 120
         var pixels = h - topInfo - btnDown;
     }
-    if(pixels < 300) {
+
+    if (pixels < 300) {
         // minimum usable height
         pixels = 300;
     }
-    p.css({ height: (pixels)+"px" });
-}
+
+    p.css({
+        height: pixels + "px"
+    });
+}
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2747543739/issues/source?bid=3176160&fileBranchId=3355598#l98)